snapcraft/cli/assertions.py

```diff
@@ -77,3 +101,87 @@ def validate(snap_name, validations, key_name):
 def gated(snap_name):
     """Get the list of snaps and revisions gating a snap."""
     snapcraft.gated(snap_name)
+
+
+@assertionscli.command('edit-collaborators')
+@click.argument('snap-name', metavar='<snap-name>')
+@click.option('--key-name', metavar='<key-name>')
+def edit_collaborators(snap_name, key_name):
+    """Edit the list of collaborators for <snap-name>."""
+    dev_assertion = assertions.DeveloperAssertion(
+        snap_name=snap_name, signing_key=key_name)
+    developers = dev_assertion.get_developers()
+    updated_developers = _update_developers(developers)
+    new_dev_assertion = dev_assertion.new_assertion(
+        developers=updated_developers)
+    if new_dev_assertion.is_equal(dev_assertion):
+        echo.warning('Aborting due to unchanged collaborators list.')
+        return
+
+    try:
+        new_dev_assertion.push()
+    except storeapi.errors.StoreValidationError as store_error:
+        if store_error.error_list[0]['code'] != 'revoked-uploads':
+            raise store_error
+        click.echo('This will revoke the following collaborators: {!r}'.format(
```

Comment:
The response really lists uploads/revisions of the snap, not collaborators, although the upload does include info about its developer.

Reply:
it would be interesting to record this